### DISCUSSION TITLE
[QUESTION] How the heck to get the ISO versions that are compatible with Sophia Script ?

### DISCUSSION THREAD

**Discussion by YHSX88** [farag2/Sophia-Script-for-Windows#295] [Q&A] [answered]
Hi 

Just trying to figure a way to get the exact version 19043.1151 of win 10 ISO that is mentioned on the home page ? 

As suggestion: maybe you can add the links to media creation tool if you have them.

Thanks in advance.

**Comment by farag2** [comment, marked as answer]
Hi, `19043.1151` is just the minimum supported Windows 10 version. There is no need to find the exact build number ISO (it doesn;t exist). Just download the latest ISO from the MS site via Media Creation Tool: <https://go.microsoft.com/fwlink/?LinkId=691209>

**Reply by YHSX88** [reply]
So as long it's **21H1** ver 5.12.5 should work ? Thanks !!

**Reply by farag2** [reply]
of course. The main criteria is the Window 10 build number. The minimum supported is 1151.

**Reply by YHSX88** [reply]
Ok just feedback ... in the youtube videos attached to the git homepage it was said make sure you get exact Win Version .. and I thought ok I'll get it but then I couldn't find it .. so this led me into thinking I need exactly 1151 .. JFYI . .. but then maybe it's just me ))

Thanks Again !

**Reply by farag2** [reply]
David will release a new video in a few days, updated and improved. I don't record or edit a video material due to I'm not a English native speaker (but know it :D). So we decided, that he would handle with this. He's voice is still monotone xD

**Reply by farag2** [reply]
Also take a look at https://github.com/Sophia-Community/SophiApp
The next step of Sophia project. Screens: https://t.me/SophiaNews/396

**Reply by YHSX88** [reply]
Mhm ok I see, JFYI your english is good, i starred the new project!

So hopefully once it's released will try it. 

Thanks

**Reply by farag2** [reply]
The deadline for the 1.0 version is December, 31. Rhis year, or course)
